packages/camera/camera/example/lib/main.dart

```diff
@@ -709,7 +709,7 @@ class _CameraExampleHomeState extends State<CameraExampleHome>
           showInSnackBar('Audio access is restricted.');
           break;
         case 'cameraPermission':
-          // Android & web only
+          // Android only
           showInSnackBar('Unknown permission error.');
```

Comment:
@camsim99 This is for all android permission exceptions before your change. After your change we can delete this exception.

Reply:
Note that Android's ongoing request error should be an "error" and not an "exception". So we should not handle them at all. Your current implementation seems fine, but please allow this PR to land first so that you can update the example code here.